server/main.lua

```diff
@@ -104,9 +104,8 @@ function DepositMoney(src, amount, pDataType, type)
 	local player = exports.qbx_core:GetPlayer(src)
 
 	if not player.PlayerData[pDataType].isboss then ExploitBan(src, 'depositMoney Exploiting') return false end
-
+	local account = player.PlayerData[pDataType].name
```

Comment:
What's this change?

Reply:
That seemed to have fixed the linting issues